api/controllers/userController.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var util = require('util');
+var db = require('../db')
+
+module.exports = {
+  usersPut: usersPut
+};
+
+function usersPut(req, res) {
+  var id = req.swagger.params.userId.value;
+  var firstName = req.swagger.params.userInfo.value.firstName;
+  var lastName = req.swagger.params.userInfo.value.lastName;
+  var birthMonth = req.swagger.params.userInfo.value.birthMonth;
+  var birthDay = req.swagger.params.userInfo.value.birthDay;
+  var birthYear = req.swagger.params.userInfo.value.birthYear;
+  var birthDate = util.format('%d-%d-%d,', birthYear, birthMonth, birthDay);
+  var email = req.swagger.params.userInfo.value.email;
+
+  if ((birthDay>0 && birthDay<32) && (birthMonth>0 && birthMonth<13) && birthYear) {
```

Comment:
`if !isNaN(Date.parse(birthDate))`

Reply:
Tested with birthYear to 0 and did not reply 400 but 200.